src/Tribe/Tickets.php

```diff
@@ -1832,6 +1769,71 @@ public static function tickets_own_stock( $post_id ) {
 			return true;
 		}
 
+		/**
+		 * Tries to make data about global stock levels and global stock-enabled ticket objects
+		 * available to frontend scripts.
+		 *
+		 * @deprecated TBD
+		 *
+		 * @param array $tickets
+		 */
+		public static function add_frontend_stock_data( array $tickets ) {
+			if ( is_admin() ) {
+				return;
+			}
+
+			/*
+			 * Add the frontend ticket form script as needed (we do this lazily since right now),
+			 * it's only required for certain combinations of event/ticket.
+			 */
+			if ( ! empty( self::$frontend_script_enqueued ) ) {
+				return;
+			}
+
+			$plugin = Tribe__Tickets__Main::instance();
+
+			wp_enqueue_script( 'wp-util' );
+
+			tribe_asset(
+				$plugin,
+				'tribe_tickets_frontend_tickets',
+				'frontend-ticket-form.js',
```

Comment:
This file is deleted - so it'll never load? Are we trying to use this just for the localized variables? (I'm fairly sure they are all loaded in `event-tickets/src/Tribe/Editor/Blocks/Tickets.php->assets()`

Or was this supposed to change to load `tickets-block.js`?

Reply:
From what I saw tickets-block has its own localize script. My QA of this did not produce any errors that it was missing something. 

I was wondering if the js file should remain until this deprecated method is removed?

Reply:
I think I should add the deprecated method to the top of the function and remove the rest of the contents so it does not try to load the missing file.

Reply:
Yeah, `tickets-block.js` does have it's own localize.

I'm down with the shift so the file doesn't load - maybe as part of deprecating it we just call for `tickets-block.js` instead?

Reply:
If you make any change, go ahead and take off the Q label 😄